server/ServerApp.cpp

```diff
@@ -1288,11 +1292,15 @@ void ServerApp::LoadGameInit(const std::vector<PlayerSaveGameData>& player_save_
         // the player controlling a particular empire might have a different
         // player ID than when the game was first created
         m_player_empire_ids[player_id] = empire_id;
+    }
 
-
+    for (const auto& psgd : player_save_game_data) {
+        int empire_id = psgd.m_empire_id;
         // add empires to turn processing, and restore saved orders and UI data or save state data
-        if (GetEmpire(empire_id)) {
-            AddEmpireTurn(empire_id);
+        Empire* empire = GetEmpire(empire_id);
```

Comment:
can put this line into the `if ( )` directly

Reply:
Fixed.